src/connector/src/sink/redis.rs

```diff
@@ -44,13 +48,57 @@ pub const VALUE_FORMAT: &str = "value_format";
 #[derive(Deserialize, Debug, Clone, WithOptions)]
 pub struct RedisCommon {
     #[serde(rename = "redis.url")]
```

Comment:
I think for single node setting, we should allow not wrapping the address with `[]` like `url="redis:6379"` to be compatible with the original way of config.